src/pybind/mgr/dashboard/tools.py

```diff
@@ -910,3 +915,52 @@ def find_object_in_list(key, value, iterable):
         if key in obj and obj[key] == value:
             return obj
     return None
+
+
+def update_dict(data, update_data):
```

Comment:
Didn't you plan to move this function to ``tests/__init__.py`` to make it use for tests only?

Reply:
Yup, that's right. I'm uncertain. It's the only thing I said I would do but haven't. I'm not sure about if `__init__.py` is the right place. I looked into that file and it would be the first helper. And there's other stuff going on there, too. Thinking about putting the function to either a `helper.py` or leave it where it is. Both option seem to fit better than `__init__.py`. What do you (or others) think about that?

Reply:
Moving it to ``helper.py`` sounds good.